lib/learn_test/strategies/none.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module LearnTest
+  module Strategies
+    class None < LearnTest::Strategy
+      def service_endpoint
+        '/e/flatiron_none'
```

Comment:
Related change for Ironbroker https://github.com/flatiron-labs/ironbroker-v2/pull/96